examples/fabric/reinforcement_learning/rl/agent.py

```diff
@@ -24,7 +23,8 @@ def __init__(self, envs: gym.vector.SyncVectorEnv, act_fun: str = "relu", ortho_
             raise ValueError("Unrecognized activation function: `act_fun` must be either `relu` or `tanh`")
         self.critic = torch.nn.Sequential(
             layer_init(
-                torch.nn.Linear(np.array(envs.single_observation_space.shape).prod(), 64), ortho_init=ortho_init
+                torch.nn.Linear(int(torch.Tensor(envs.single_observation_space.shape).prod()), 64),
```

Comment:
same here, we should always use `torch.tensor()` in favor of torch.Tensor(). I know it's weird but it's what torch recommends.